pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -297,24 +301,30 @@ const OSRow = ({ vendor, osInfoList, os, vendors, onValueChanged, validationFail
     );
 };
 
-const MemoryRow = ({ memorySize, memorySizeUnit, nodeMaxMemory, onValueChanged }) => {
+const MemoryRow = ({ memorySize, memorySizeUnit, nodeMaxMemory, onValueChanged, validationFailed }) => {
+    const validationStateMemory = validationFailed.memory ? 'error' : undefined;
+
     return (
         <React.Fragment>
             <label htmlFor='memory-size' className='control-label'>
                 {_("Memory")}
             </label>
-            <FormGroup bsClass='ct-validation-wrapper' controlId='memory'>
+            <FormGroup validationState={validationStateMemory} bsClass='form-group ct-validation-wrapper' controlId='memory'>
```

Comment:
You don't need to add form-group class, it should be added by the FormGroup patternfly component, no?

Reply:
It's weird, but it seems once you define your own bsClass, you lost form-group added by Patternfly.
e.g. Name Row doesn't specify it's own bsClass, and it does have form-group added. Storage Size row specified it's own bsClass, and it looses form-group.

Reply:
Right, noticed just. I just tried to add it and I don't see any visual change but since we have it everywhere let's keep it here.